packages/opentelemetry-api/src/api/trace.ts

```diff
@@ -15,15 +15,17 @@
  */
 
 import { NOOP_TRACER_PROVIDER } from '../trace/NoopTracerProvider';
-import { TracerProvider } from '../trace/tracer_provider';
 import { Tracer } from '../trace/tracer';
+import { TracerProvider } from '../trace/tracer_provider';
+import { GLOBAL_TRACE_API_KEY, makeGetter, _global } from './global-utils';
+
+const API_VERSION = 0;
```

Comment:
should this be shared with `propagations.ts` ?

Reply:
Right now each top-level API gets its own version. That way we can bump one without having to bump others. If you don't like this we can talk about it.

Reply:
I just don't understand why is it like this, maybe you can add some doc explanation - it will help to understand it better and how this should be then bumped and when

Reply:
I decided it is simpler if they just all share a single version. Check out https://github.com/open-telemetry/opentelemetry-js/pull/943/commits/a01c20a98da3fdea963bd286f918a7d86c127885 and lmk what you think.